application/deed/searchdeed/views.py

```diff
@@ -68,27 +68,67 @@ def show_authentication_code_page():
     return render_page
 
 
-def verify_auth_code(auth_code):
+@searchdeed.route('/confirming-mortgage-deed', methods=['POST'])
+def show_confirming_deed_page():
+
+    auth_code = request.form['auth_code']
 
     if auth_code is None or auth_code == '':
         return render_template('authentication-code.html', error=True)
 
-    deed_api_client = getattr(searchdeed, 'deed_api_client')
-    response = deed_api_client.verify_auth_code(str(session.get('deed_token')),
-                                                str(session.get('borrower_token')),
-                                                auth_code)
+    page_render = render_template('confirming-mortgage-deed.html', auth_code=request.form['auth_code'])
```

Comment:
use of a variable is redundant here as it is only used once the render template command can just be directly plugged into the return